README.md

```
@@ -11,13 +11,17 @@ See also: https://github.com/pre-commit/pre-commit
+    rev: v0.2.0  # Use the ref you want to point at
     hooks:
     -   id: check-import-datetime
+    -   id: check-import-unittest-testcase
 ```
 
 ### Hooks available
 
 #### `check-import-datetime`
 Checks that datetime module is not imported directly.
+
+#### `check-import-unittest-testcase`
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)